### ISSUE TITLE
UITK Formatted Input A11y Review

### ISSUE THREAD

**Issue by jake-costa** [jpmorganchase/salt-ds#5556]
A11y review/feedback on the UITK implementation, which uses a mask, and a 3rd party library for formatting. 

This information will assist the Salt team on writing a migration guide explaining what the Salt equivalent of UITK's Formatted Input is/should be and what to consider when using it.
